testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -2755,6 +2755,9 @@ public void listAllTypes() throws IOException {
     List<Map<String, String>> fields = (List<Map<String, String>>) response.get(0).get("fields");
     assertThat(fields.size()).isEqualTo(1);
     assertThat(fields.get(0).get("name")).isEqualTo("firstname");
+
+    // 20-Dec-2021, tatu: This is due to [stargate#1499], so fails with v2.0.0
```

Comment:
@ivansenic As per my comment, leaving test as-is, until SGv1 part can be addressed -- or we change behavior within v2.0.0.